Activity.js

```diff
@@ -45,20 +45,21 @@ class Activity {
 
   markComplete() {
     this.completed = true;
-    pastActivities.push(this);
-    console.log(pastActivities);
+    // pastActivities.push(this);
```

Comment:
**Opinion that probably doesn't apply:** In general I don't comment out code - it's either in the codebase or it's deleted. In your case, since this whole thing is in progress, there's good reason to leave code in you might want for later.

Reply:
**Opinion:** Leaving commented code in the codebase makes things cluttered

Reply:
Excellent. We removed a lot of code as we move to a final state this evening and will do more tomorrow. Thanks so much!